sigle/src/modules/editor/EditorToolbar/ToolbarBubbleMenu.tsx

```diff
@@ -0,0 +1,171 @@
+import { useState } from 'react';
+import { Editor } from '@tiptap/react';
+import {
+  Link1Icon,
+  FontBoldIcon,
+  FontItalicIcon,
+  CodeIcon,
+  Cross2Icon,
+  StrikethroughIcon,
+  UnderlineIcon,
+} from '@radix-ui/react-icons';
+import { styled } from '../../../stitches.config';
+import { Flex } from '../../../ui';
+
+const BubbleMenuButton = styled('button', {
+  color: '$gray11',
+
+  variants: {
+    active: {
+      true: {
+        color: '$orange10',
+      },
+    },
+  },
+});
+
+const BubbleMenuInput = styled('input', {
+  width: '100%',
+  pl: '$2',
+  pr: '$1',
+  backgroundColor: 'transparent',
+  outline: 'none',
+});
+
+// Maybe can be used on clicks https://github.com/ueberdosis/tiptap/issues/104#issuecomment-912794709
+
+interface BubbleMenuProps {
+  editor: Editor;
+}
+
+export const ToolbarBubbleMenu = ({ editor }: BubbleMenuProps) => {
```

Comment:
Wdyt of about calling this component `ToolbarMenu` instead of ToolbarBubbleMenu?

Reply:
yeah that makes more sense :)